htdocs/core/modules/fichinter/modules_fichinter.php

```diff
@@ -54,18 +54,18 @@ static function liste_modeles($db, $maxfilenamelength = 0)
 		global $conf;
 
 		$type='ficheinter';
-		$liste=array();
+		$list=array();
 
 		include_once DOL_DOCUMENT_ROOT.'/core/lib/functions2.lib.php';
-		$liste=getListOfModels($db, $type, $maxfilenamelength);
+		$list=getListOfModels($db, $type, $maxfilenamelength);
 
-		return $liste;
+		return $list;
 	}
 }
 
 
 /**
- *  Classe mere des modeles de numerotation des references de fiches d'intervention
+ *  Mother class numbering models of intervention sheet references
```

Comment:
Wouldnt "parent class"  be the better translation for "class de mere" as you used above ?
Other possibilty would be "Base class"

Reply:
You're right, I did it line 26 before and not here :)